adapter/src/dbt/adapters/fal_experimental/adapter.py

```diff
@@ -71,8 +76,32 @@ def run_in_environment_with_adapter(
     else:
         deps = get_default_pip_dependencies()
         stage = VirtualPythonEnvironment(deps)
+        fal_scripts_path = _get_fal_scripts_path(config)
 
-        with PythonIPC(environment, environment.create(), extra_inheritance_paths=[stage.create()]) as connection:
+        with PythonIPC(environment, environment.create(), extra_inheritance_paths=[fal_scripts_path, stage.create()]) as connection:
             execute_model = partial(_isolated_runner, code, config, manifest, macro_manifest)
             result = connection.run(execute_model)
             return result
+
+@contextmanager
+def extra_path(path: str):
+    import sys
+    sys.path.append(path)
+    try:
+        yield
+    finally:
+        sys.path.remove(path)
+
+def _get_fal_scripts_path(config: RuntimeConfig):
+    import pathlib
+    cwp = pathlib.Path.cwd()
+
+    fal_scripts_path = config.project_root
```

Comment:
Is `project_root` a sane default for script paths?

Reply:
This is what we had for fal cli and what we have documented for it. But it is a good question! We could just NOT include anything if not specified.